static/js/thing.js

```diff
@@ -507,6 +514,28 @@ class Thing {
         3000);
     }
   }
+
+  /**
+   * Handle a 'connected' message.
+   * @param {boolean} connected - New connectivity state
+   */
+  onConnected(connected) {
+    this.connected = connected;
+
+    const opacity = connected ? 1 : 0.5;
+
+    if (this.format === Constants.ThingFormat.EXPANDED) {
+      this.layout.svg.style.opacity = opacity;
```

Comment:
The opacity modification would be cleaner done with the addition of a class so that when seeing the 0.5 opacity in the CSS I can refer to the same file to see the requirements for the opacity to be set to 1

Reply:
Done.